### ISSUE TITLE
Change default border

### ISSUE THREAD

**Issue by pat-schmitt** [OGGM/oggm#1495]
The current default prepro border is 40, but this option is not available in the current v1.6 gdirs https://cluster.klima.uni-bremen.de/~oggm/gdirs/oggm_v1.6/L3-L5_files/elev_bands/w5e5/qc0/pcpwin/match_geod_pergla/RGI62/.

Maybe we should change the default border to 80 or better 160 when thinking of the potential use of the dynamic mu star calibration as a future option (because the dynamic calibration works for more glaciers if a larger border is used).

**Comment by fmaussion** [comment]
Yes, or similar to other recent changes: remove any default and make it explicit.
